test/In.ProjectEKA.HipServiceTest/DataFlow/Builder/TestBuilder.cs

```diff
@@ -44,5 +55,24 @@ internal static Consent Consent()
                 ConsentStatus.GRANTED
             );
         }
+        
+        internal static DataRequestLib DataFlowRequest()
+        {
+            var faker = new Faker();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename 'faker' which hides the field with the same name.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5073772)